aten/src/ATen/record_function.cpp

```diff
@@ -328,6 +330,14 @@ void RecordFunction::before(std::string name, int64_t sequence_nr) {
   manager().runStartCallbacks(*this);
 }
 
+/* static */ void RecordFunction::setDefaultNodeId(int64_t newDefaultNodeId) {
+  defaultNodeId = newDefaultNodeId;
```

Comment:
ok, so as I understand from the offline discussion, this in principle needs to happen only once per init_rpc

Reply:
also, do we need to check that newDefaultNodeId>=0?

Reply:
Yes, this is only called in init_rpc now, not every RPC call, and there should be no changes to `record_function` ctx mgr.